pkg/sqlquery/config.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package sqlqueryreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlqueryreceiver"
+package sqlquery // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/sqlquery"
```

Comment:
How confident are you that we need to move the receiver config to be part of this library? I feel like the config is a receiver-only thing, and the sql API shouldn't know about it.

Reply:
I think I need to redo this work, based off what I learned from oracledb. This PR is not good enough.